eZ/Bundle/EzPublishRestBundle/Resources/config/value_object_visitors.yml

```diff
@@ -159,35 +159,35 @@ services:
     ezpublish_rest.output.value_object_visitor.InvalidArgumentException:
         parent: ezpublish_rest.output.value_object_visitor.base
         class: %ezpublish_rest.output.value_object_visitor.InvalidArgumentException.class%
-        arguments: [ %kernel.debug% ]
+        arguments: [ %kernel.debug%, @translator ]
```

Comment:
Should we add a base visitor for exceptions, that injects the translator ? We will add more over time.